src/config/dynamic-theme-fixes.config

```diff
@@ -9722,6 +9722,19 @@ IGNORE INLINE STYLE
 
 ================================
 
+inet.se
+
+INVERT
+.h1yh8nho
+.i1gv55ha
```

Comment:
Why does this element needs to be inverted?

Reply:
It's the icons for the different categories in menus on the site, that are black if not inverted which makes them difficult to see.

Reply:
```suggestion
a.item-button span[class*="icon"]
```

Reply:
That works better, thanks! Still find that when turning on darkmode in dark reader some icon in main side menu first loads in correctly to then unload and be replaced by none or broken icon elements (was a problem before making corrections to dark theme)